sensorimotor/experiments/capacity/run.py

```diff
@@ -69,12 +69,14 @@ def run(args):
   numWorlds = defn["worlds"]
   numElements = defn["elements"]
 
-  filePrefix = os.path.join(outputDir, "{0}x{1}".format(numWorlds,
```

Comment:
If you instead use the format flag to pre-fill the two numbers with `0`s, you can avoid the complicated logic below.